ci/qs-test/taskcat_overlay.yml

```diff
@@ -43,7 +43,7 @@ project:
     TanzuNetRelocateImages: "No"
     TAPDomainName: #@ "{}.{}".format("$[taskcat_random-string]", domain)
     QSS3BucketName: #@ bucket + "-" + bucketRegion
-    QSS3BucketRegion: #@ bucketRegion
+    QSS3BucketRegion:  $[taskcat_current_region]
```

Comment:
As discussed on slack: This is only part of the change we got to do. This change will break other parts of the pipeline.

Reply:
To be clear about what I mean: [see here](https://runway-ci-sfo.eng.vmware.com/teams/tappc/pipelines/qs-test/jobs/prepare-bucket/builds/51#L63f6a68f:1911:1922)
In this instance it doesn't matter, it's just printing garbage, however in the `publish-bucket` job, it would fail, because there is no region `$[taskcat_current_region]`